datasets/am/hetq/crawler.py

```diff
@@ -0,0 +1,298 @@
+    position_name = position_en or position_hy
+    context.log.debug(f"position: {position_name} years: {data['years']}")
+    if not position_name:
+        context.log.warning(f"Skipping person {person_id} with no position")
+        return None
+    position = h.make_position(
+        context,
+        name=position_name,
+        country="am",
+    )
+    categorisation = categorise(context, position, is_pep=True)
+    if not categorisation.is_pep:
+        context.log.warning(f"Person {person_id} is not PEP")
+        return None
+    occupancy = h.make_occupancy(
+        context,
+        person,
+        position,
+        start_date=str(min(data["years"])),
+        # skip this for now as it's not an explicit end date, and we
+        # get basically no PEPs as the data is old
+        # end_date=str(max(data["years"]) + 1),
+        categorisation=categorisation,
+    )
+    context.log.debug(f"categorization {categorisation} occupancy {occupancy}")
+    if occupancy is not None:
+        context.emit(person, target=True)
+        context.emit(position)
+        context.emit(occupancy)
+        return person
+    context.log.info(f"No entities emitted for {person_id} ({name_en} / {name_hy})")
+    return None
+
+
+def crawl_list(
+    context: Context,
+    peps: Dict[int, Dict[str, Any]],
+    data: List[Dict[str, Any]],
+    year: int,
+    lang: SupportedLanguage,
+):
+    """Accumulate person info from yearly list.  Do not actually
+    create entities yet because they might not be PEPs."""
+    for entry in data:
+        personID = entry.pop("personID")
+        fullName = entry.pop("fullName")
+        lastPosition = entry.pop("lastPosition")
+        context.audit_data(entry, UNUSED_FIELDS)
+        context.log.debug(
+            f"Found person {personID} ({fullName}, {lastPosition}) in {year}"
+        )
+        if personID not in peps:
+            peps[personID] = {
+                f"name_{lang}": fullName,
+                f"position_{lang}": lastPosition,
+                "years": {year},
+            }
+        else:
+            pep = peps[personID]
+            pep["years"].add(year)
+            if f"name_{lang}" not in pep:
+                pep[f"name_{lang}"] = fullName
+            else:
+                assert pep[f"name_{lang}"] == fullName
+            # NOTE: One might expect there to be different positions
+            # in different years but this is not the case
+            if f"position_{lang}" not in pep:
+                pep[f"position_{lang}"] = lastPosition
+            else:
+                assert pep[f"position_{lang}"] == lastPosition
+
+
+def crawl_relations(
+    context: Context, zipfh: ZipFile, peps: Dict[int, Dict[str, Any]], person_id: int
+):
+    """Read relation graph and create entities."""
+    if person_id not in peps or "entity" not in peps[person_id]:
+        context.log.debug(f"Skipping relations for {person_id}: is not PEP")
+        return
```

Comment:
let's keep them due to the decision to emit all as `poi`

Reply:
Ah - actually the log message here (and the variable name `peps`) is misleading.  We need an entity in order to make a relation, that's all.   So if we keep and emit all the Person entities then we'll do the right thing here by default.